a10_octavia/common/utils.py

```diff
@@ -53,6 +53,7 @@ def validate_partial_ipv4(address):
     if not netaddr.valid_ipv4('.'.join(octets), netaddr.core.INET_PTON):
         raise cfg.ConfigFileValueError('Invalid partial IPAddress value given'
                                        ' in configuration: {0}'.format(address))
+    return address
```

Comment:
May need to check if existing UTs passes - `tests/unit/common/test_utils.py`  for this change.

Reply:
Done. Had missed this out and CI tests do not seem to be running with this PR. Manually verified all unit tests passing.